lib/users/index.js

```diff
@@ -33,13 +34,21 @@ exports.create = function(options) {
     return utils.delete(_.extend(optionsToSend, deleteOptions), callback);
   };
 
-  return {
-    getUser                : listAllUsers,
-    listAllUsers           : listAllUsers,
-    getCurrentUser         : getCurrentUser,
-    addUser                : addUser,
-    addUserAndSendEmail    : addUserAndSendEmail,
-    updateUser             : updateUser,
-    removeUser             : removeUser
+  var buildUserObject = function () {
+    var userObject = {
+      getUser                : listAllUsers,
```

Comment:
This is an odd one.  Is there an API function for getting a user by ID?

Reply:
There are two endpoints that are the same with the exception of the trailing id. If the id is specified, Get User is called. If it is not specified, Get All Users is called

Reply:
Gotcha.  Mostly looks weird, but probably harmless.